azure-pipelines.yml

```diff
@@ -1,88 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should this file eventually live in the `pipelines` folder?

Reply:
yes, moved the file to pipelines folder.